web/stories/ui/formatted-text.stories.svelte

```diff
@@ -0,0 +1,201 @@
+
+  let longHeadingText =
+    'This is a very long heading text that goes on and on. It is meant to test the rendering of long headings in the UI. It includes many words and phrases to ensure that the heading wraps correctly and maintains its styling.'
+
+  let longHeadings = `<h1>${longHeadingText}</h1><h2><a href="#">${longHeadingText}</a></h2><h3>${longHeadingText}</h3><h5>${longHeadingText}</h5><h6>${longHeadingText}</h6>`
+
+  let headingsWithParagraphs = `<h1>Heading H1</h1>${paragraph}${paragraph}<h2>Heading H2</h2><p>${paragraph}${paragraph}${paragraph}</p><h3>Heading H3</h3><p>${paragraph}</p><h4>Heading H4</h4><p>${paragraph}</p><h5>Heading H5</h5><p>${paragraph}</p><h6>Heading H6</h6><p>${paragraph}</p>`
+
+  let images =
+    '<img src="https://media.dev.to/cdn-cgi/image/width=800%2Cheight=%2Cfit=scale-down%2Cgravity=auto%2Cformat=auto/https%3A%2F%2Fdev-to-uploads.s3.amazonaws.com%2Fuploads%2Farticles%2Fcl51qc17mia9ep2ujcdh.gif" alt="A scene from ..." loading="lazy" width="480" height="274" ><img src="https://media.dev.to/cdn-cgi/image/width=1000,height=420,fit=cover,gravity=auto,format=auto/https%3A%2F%2Fdev-to-uploads.s3.amazonaws.com%2Fuploads%2Farticles%2Fwjyxxca8qxz6j0vc7av9.jpg" style="aspect-ratio: auto 1000 / 420;" width="1000" height="420"  alt="Cover">'
+
+  let inlineTagsExample =
+    '  <p><b>Bold</b> Text Example</p>\n' +
+    '  <p><i>Introduction</i></p>\n' +
+    '  <p><strong>Fish</strong> are a group of <em>aquatic</em> animals that first appeared during the <mark>Devonian</mark> period.</p>\n' +
+    '  <p><bdi>Types of Fish</bdi></p>\n' +
+    '  <p><bdo>Characteristics of Fish</bdo></p>\n' +
+    '  <p>Fish are characterized by their <strong>gills</strong>, <em>fins</em>, and <mark>scales</mark>.</p>\n' +
+    '  <p><cite>Interesting Facts</cite></p>\n' +
+    '  <p>Did you know that the <data value="whale-shark">whale shark</data> is the largest fish in the world?</p>\n' +
+    '  <p><del>Conclusion</del></p>\n' +
+    '  <p><ins>Fish are fascinating creatures!</ins> They come in all shapes and sizes, from the tiny <small>paedocypris</small> to the massive <samp>whale shark</samp>.</p>\n' +
+    '  <p><sub>Subscript text</sub> and <sup>superscript text</sup> can also be used for scientific notation.</p>\n' +
+    '  <p>For more information, visit our <a href="https://example.com">website</a>.</p>\n'
+
+  let unorderedList =
+    '<ul>\n' +
+    '  <li>Simple list item</li>\n' +
+    '  <li><a href="https://example.com">Item with link</a></li>\n' +
+    '  <li><strong>Strong list item</strong></li>\n' +
+    '  <li>Item <i>with itallic formatt</i></li>\n' +
+    '</ul>'
+
+  let orderedList = unorderedList.replaceAll('ul', 'ol')
+
+  let contentTags =
+    '  <figure>\n' +
+    '    <img src="https://media.dev.to/cdn-cgi/image/width=400,height=220,fit=cover,gravity=auto,format=auto/https%3A%2F%2Fdev-to-uploads.s3.amazonaws.com%2Fuploads%2Farticles%2F7u4wo1pru5qle7jj67of.png" alt="Example image">\n' +
```

Comment:
You can use images from `web/public/`. It will be more reliable and works in offline.